services/backend/src/service/basin_subzone.js

```diff
@@ -21,7 +21,7 @@ module.exports = (basinSubzonePersistence, seService) => {
     /**
      * Get subzone total area divided by strategic ecosystem type
      */
-    getAreaBySE: async (subzoneId) => {
+    /* getAreaBySE: async (subzoneId) => {
```

Comment:
Why did you commented instead of delete?

Reply:
Deleted!